engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -511,6 +532,16 @@ public JSONObject getLogicalOperators() throws Exception
     nodeList.operators = dagManager.getLogicalOperatorInfoList();
     return new JSONObject(objectMapper.writeValueAsString(nodeList));
   }
+  
+  @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES)
+  @Produces(MediaType.APPLICATION_JSON)
+  public JSONObject getLogicalModules(@QueryParam("flatten") String flatten) throws Exception
```

Comment:
Try using BooleanUtils.isTrue() on the Boolean object, and pass ?flatten=true to the URL